frontend/src/App.js

```diff
@@ -1,25 +1,29 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import SignUp from './SignUp';
+
 
 function App() {
+
+  // const [count, setCount] = useState(0);
+  // function handleClick() {
+  //   setCount(count + 1);
+  // }
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+
+    <div className='App'>
+      <header className="header">
+
+      {/* <h1> Welcome to my app</h1> */}
+      <SignUp  />
+      
```

Comment:
@joonkyu this may result in a merge conflict because you have to link all the components on the app.js file, you two should talk it through